### PR TITLE
Added Wildlife Park 2

### PR DESCRIPTION
**Pull request by Zemogiter**

### CODE REVIEW

File: Applications/Games/Wildlife Park 2/Local/script.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+new LocalInstallerScript()
+    .name("Wildlife Park 2")
+    .editor("B-Alive")
+    .applicationHomepage("www.wildlifepark2.com")
+    .author("Zemogiter")
+    .category("Games")
+    .executable("WLP2.exe")
+    .preInstall(function(wine,wizzard) {
+	wizard.message(tr("On first run the game might not go into full screen. If that happens go to options and set the resolution to 1280x960. You will be asked to close the game in order to apply the new settings. Click Yes. Once you starte the game again you should see a window when you can set your game resolution to match your screen."));
```
Comment (plata) [inline Applications/Games/Wildlife Park 2/Local/script.js]: typo: "starte"

"where you can set"

we need #361 ...

**Comment by plata** [comment]
The typo is in the other script as well.

**Comment by plata** [comment]
I don't understand what's wrong with codacy.

**Comment by Zemogiter** [comment]
Me neither. I have just tested the local script (I don't have steam version of this game but should work) and it works just fine (both the installation and gameplay). The lines codacy is complaining about are the same across every script.

File: Applications/Games/Wildlife Park 2/Steam/script.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+include(["Engines", "Wine", "QuickScript", "SteamScript"]);
+include(["Engines", "Wine", "Verbs", "quartz"]);
+
+new SteamScript()
+    .name("Wildlife Park 2")
+    .editor("B-Alive")
+    .applicationHomepage("www.wildlifepark2.com")
+    .author("Zemogiter")
+    .appId(264710)
+    .executable("Steam.exe", ["-silent", "-applaunch", 304350, "-no-ces-sandbox"])
```
Comment (plata) [inline Applications/Games/Wildlife Park 2/Steam/script.js]: probably: "-no-cef-sandbox"

However: that's default so this line is not needed.

Comment (Zemogiter) [reply Applications/Games/Wildlife Park 2/Steam/script.js]: The "-no-ces-sandbox" argument is used in my Subnautica script and codacy didn't threw anything about it.

**Comment by plata** [comment]
I think you misunderstood me: you can remove the **complete** line not just "no-cef-sandbox".

File: Applications/Games/Wildlife Park 2/Steam/script.js (hunk 2 above)
Comment (plata) [reply Applications/Games/Wildlife Park 2/Steam/script.js]: Codacy cannot know. It's just a string, you could write anything there.

**Pull request merged**